examples/svelte-apollo/src/apollo.js

```diff
@@ -0,0 +1,44 @@
+import ApolloClient from "apollo-client";
+import { InMemoryCache } from "apollo-cache-inmemory";
+import { WebSocketLink } from "apollo-link-ws";
+import { split } from "apollo-link";
+import { HttpLink } from "apollo-link-http";
+import { getMainDefinition } from "apollo-utilities";
+
+const headers = {'content-type': 'application/json','x-hasura-admin-secret': process.env.Hasura_Secret};
+const getHeaders = () => {
+  return headers;
+};
```

Comment:
We should not be using the `x-hasura-admin-secret` here in the client (very insecure at it gives everyone admin access to the whole GraphQL API).

We should instead use the `Auhtorization` header and the access token generated by `nhost.auth.getAccessToken()`. If no access token is available, meaning the user is not signed in, no `Authorization` header should be set.

Reply:
I'll take a look to that! But here I have saved the ``x-hasura-admin-secret`` in env, to not give access to everyone to my GraphQL API. I'll make the necessary changes and update the PR soon.

Reply:
Isn't this whole Svelt app running client-side? If it is, the env var is exposed client-side? Or am I missing something?

Reply:
Yeah, its client side app. I've made the changes using nhost.auth.getAccessToken() just need to check it a bit. For env part i meant like it's in .gitignore so no one will able to access it ant any point. I had one small doubt like here i passed headers, how to do same with nhost.auth.getAccessToken() ? Any snippet or references for that?

Reply:
> Yeah, its client side app. I've made the changes using nhost.auth.getAccessToken() just need to check it a bit.

Great. 👍 

> For env part i meant like it's in .gitignore so no one will able to access it ant any point.

Putting the `.env` file in `.gitignore` does not mean that the admin secret is not exposed client side? This Svelt example, from my understanding, is still a client-side application that has the admin secret in the app, client-side when running. Meaning, that anyone using the app is able to also read the admin secret and use the admin secret to get full access to Hasura.

Reply:
I think so like as of now when using in react or other frameworks i save all secrets in ``.env`` and then keep it in ``.gitignore``.  Apps can't use then admin secret in such conditions, if required to use such while deployment whatever platform is being used in there env variable i use to keep such secret so that again no one is able to use these for any other purposes.